### DISCUSSION TITLE
GitHub Container Registry points to incorrect README

### DISCUSSION THREAD

**Discussion by xezpeleta** [community/community#25665] [Actions]
<p>Hi!</p>
<p>I’ve had published a Docker image to <a href="http://ghcr.io" rel="noopener nofollow ugc">ghcr.io</a> and I see it correctly linked to it’s repository, under “Packages”.</p>

<p>However, it tooks the repo’s main README ("/README.md") instead of the right one (“docker/README.md”). Do you know if is there a way to set a custom path for this README?</p>
<p>Thanks</p>

**Comment by clarkbw** [comment, marked as answer]
<aside class="quote no-group" data-username="xezpeleta" data-post="1" data-topic="176437">
<div class="title">
<div class="quote-controls"></div>
 xezpeleta:</div>
<blockquote>
<p>Do you know if is there a way to set a custom path for this README?</p>
</blockquote>
</aside>
<p>There’s no way to do this right now but we plan to enable this in the future. We’ll likely look for the documentation metadata label when we support this.</p>

**Reply by kij** [reply]
Just got bitten by this.  Is there a way now?
Because cutting the folder path from the org.opencontainers.image.source is unexpected.

**Reply by theanurin** [reply]
Any progress on that?

**Reply by flobernd** [reply]
Any news regarding this feature? @clarkbw

**Reply by kij** [reply]
My solution has been to `git subtree --prefix=dockerimages/subfolder -b branchname-with-new-root` and push that to a new repo.

**Comment by xezpeleta** [comment]
<blockquote>
<p>There’s no way to do this right now but we plan to enable this in the future. We’ll likely look for the documentation metadata label when we support this.</p>
</blockquote>
<p>That would be great! Thanks for the answer</p>

**Comment by rhhamburg** [comment]
<p>Hello <a class="mention" href="/clarkbw">@clarkbw</a>, is there any news on this subject?<br>
Is there a public feature request or something similar that you could follow to be kept informed about updates on this topic?</p>

**Comment by Niek** [comment]
<p>Same problem here, it would be great to link to a different README using the <code>org.opencontainers.image.documentation</code> label.</p>

**Comment by itrich** [comment]
I'm facing the same issue and wanted to ask whether there is a possibility to customize the `README.MD` for a package on ghcr.io?

**Comment by camronkhan** [comment]
+1

**Comment by evantill** [comment]
Example of usecase : mono repository pattern

**Comment by carlosperate** [comment]
I assume there has been no progress on this yet, but apart from that, it looks like this discussion is tagged with the `Actions` instead of the `Packages` label, can any of the moderators update it?

**Comment by trajan0x** [comment]
Would love to see this, we have 22 packages [in this repo](https://github.com/synapsecns/sanguine) and it's very confusing in a monorepo setup for users

**Comment by owenthereal** [comment]
Just ran into this. My use case is to not show the `README.md` at all for the Docker image. It would be nice to be able to configure this

**Comment by tropicrainforest** [comment]
@clarkbw seems to have left @github. Given that I ran across this once more, how can we revive this conversation? Or anyone here found any solution for it so far?
I reopened another [discussion](https://github.com/orgs/community/discussions/151904#discussioncomment-12262351) to increase the chance, let the @github dudes pick it over.
